util/msg/group.js

```diff
@@ -156,8 +158,12 @@ function getMembersList(group, members) {
         const footer = `${group.validated ? 'TERMINÉ - ' : ''}Dernière modif. ${moment().format('ddd Do MMM HH:mm')}`
         
         editMsgEmbed.setFooter({ text: `${footer}`});
-    
+
         await msg.edit({embeds: [editMsgEmbed]});
+
+        // "maj" component row
+      const row = await createRowGroupButtons(group);
+      await msg.edit({components: [row]})
```

Comment:
Eviter un edit en plus
```suggestion
        // "maj" component row
        const row = await createRowGroupButtons(group);
        
        await msg.edit({embeds: [editMsgEmbed], components: [row]});
```